packages/docusaurus-theme-search-algolia/src/theme/SearchBar/index.js

```diff
@@ -104,10 +101,8 @@ const Search = props => {
           {'search-bar-expanded': props.isSearchBarExpanded},
           {'search-bar': !props.isSearchBarExpanded},
         )}
-        onClick={loadAlgolia.bind(this, true)}
-        onMouseOver={loadAlgolia.bind(this, false)}
```

Comment:
Why remove `onMouseOver`? We want this as an optimization for desktop where the code starts loading as soon as the cursor hovers over.

Reply:
I get it, fixed :heavy_check_mark: